src/Carbon/Lang/zh.php

```diff
@@ -18,7 +18,7 @@
  */
 return array(
     'year'      => ':count年',
-    'month'     => ':count月',
+    'month'     => ':count个月',
```

Comment:
I totally trust you @lndj !
Can you apply https://styleci.io/analyses/q1bPvv
And squash you commits ?